client/my-sites/customer-home/cards/tasks/deprecate-editor/index.jsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'lib/formatting';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import Task from '../task';
+
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import {
+	composeAnalytics,
+	recordGoogleEvent,
+	recordTracksEvent,
+	withAnalytics,
+	bumpStat,
+} from 'state/analytics/actions';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import blockEditorImage from 'assets/images/illustrations/block-editor-fade.svg';
+import FormattedDate from 'components/formatted-date';
+import { localizeUrl } from 'lib/i18n-utils';
+import InlineSupportLink from 'components/inline-support-link';
+
+const DeprecateEditor = ( { siteId, gutenbergUrl, optIn } ) => {
+	const translate = useTranslate();
+	const actionCallback = () => {
+		optIn( siteId, gutenbergUrl );
+	};
+	return (
+		<Task
+			title={ translate( 'The new WordPress editor is coming.' ) }
+			description={ preventWidows(
+				translate(
+					'Get a head start before we activate it for everyone on {{date/}}. {{support}}Read more{{/support}}.',
+					{
+						components: {
+							date: (
+								<strong>
+									<FormattedDate date="2020-06-01" format="MMMM D" />
+								</strong>
+							),
+							support: (
+								<InlineSupportLink
+									supportPostId={ 165338 }
+									supportLink={ localizeUrl(
+										'https://wordpress.com/support/block-editor-is-coming'
+									) }
+									showIcon={ false }
+									tracksEvent="calypso_customer_home_editor_deprecate_support_page_view"
+									statsGroup="calypso_customer_home"
+									statsName="editor_deprecate_learn_more"
+								/>
+							),
+						},
+					}
+				)
+			) }
+			actionText={ translate( 'Use the WordPress editor' ) }
+			actionOnClick={ actionCallback }
+			illustration={ blockEditorImage }
+			badgeText={ translate( "What's new" ) }
+			taskId="editor-deprecation"
+			enableSkipOptions={ false }
```

Comment:
Just double-checking that we don't want to give user the "remind me" options?..

Reply:
@davemart-in, @jancavan - can you please confirm that we just want the dismiss option for this card and not the "remind me" option that is available for these tasks cards?

Reply:
That's correct. Thanks @glendaviesnz!